src/Engine/ProtoCore/ExtendedLibraries/FunctionObject.ds

```diff
@@ -1,3 +1,5 @@
+
+[InternalClass]
```

Comment:
Yes this feature supports both DS classes and functions. This is possible as a separate task as it also involved creating an IgnoreFunctionAttribute. There should be minimal cost to performance as it only occurs at compile time - when the DS code is compiled